### ISSUE TITLE
Add index page that tells users what meilix is and how meilix generator could be useful

### ISSUE THREAD

**Issue by rahulkumaran** [fossasia/meilix-generator#298]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
The website directly takes us to the page that has the form with no information about what meilix or meilix generator is and what it does.

**Expected Behaviour**
Would be great to have an index page that tells people what meilix is and how meilix generator can be used. We could redirect the users on to the form through the route `/build-iso` or something like that wherein they fill the form and build their own ISOs.

**Steps to reproduce it**
Just visit the website. Directly takes us to the page with form.

**Other information:**

All we need to do now is create a new page and transfer the form code on to that page and then replace the index.html page with basic description about meilix and meilix generator.
In the app.py file we'll have to add a new route.

**Would you like to work on the issue?**
Yes, I'd like to work on this.

**Comment by sarnava1** [comment]
@rahulkumaran yes this issue was previous raised by me. If you look at issue #280 and #285 then it tries to introduce some info page about meilix. Even I proposed a new site wherein there will be many new features and about meilix. @mariobehling approved of that project and our mentor @tabesin @meets2tarun and @xeon-zolt are already discussing about the issues. You can see the gutter channel for more info about that project. Let us discuss and maybe we can come out with a solid idea wherein everyone can contribute and that will be a good project for gci too. What say @mariobehling @meets2tarun @tabesin @xeon-zolt @abishekvashok ?

**Comment by rahulkumaran** [comment]
Right! Just saw the Gitter channel and all too. Makes sense

Just one thing though. So I'm not sure if you guys are thinking about creating a new website entirely. But IMO, we could use the existing website and just add one good landing page. Basically it would be about creating a proper index.html with all the information about the project and other necessary info that could help first time contributors and users.

**Comment by rpotter12** [comment]
we can put a link of readme.md of meilix in meilix-generator webapp. This can also be the alternative for this.

**Comment by rahulkumaran** [comment]
@rpotter12 could be an alternative but it's better if we don't redirect users to another link just to find out more about the project. People like everything to be at one place when they're using it. Just my opinion though

**Comment by sarnava1** [comment]
Yes that's why I proposed issues #280 and #285 wherein we can use the existing structure to add relevant info to the page. Since it is a web app I think a altogether new landing page is not required and like susper we can add the links in the navbar and footer. Just my opinion though. You can visit those issues and see the relevant discussions over there where @meets2tarun and @tabesin gave their respective point of views.

**Comment by pranav1698** [comment]
If we add new pages in the list and still use flask that is using normal html, css, for frontend we would be routing our users to different pages with the help of `server sided routing`; that is pretty old fashioned in terms of present web development standards; I think that's why @mariobehling suggested react or ember for the frontend beacause of the use of client side routing adding that dynamic feature in the web app;

**Comment by meets2tarun** [comment]
FOSSASIA usually wants their webapp (meilix, open-event, badgeyay) to look similar.
This idea will make it look different from others but seems interesting 
cc: @fossasia/linux-admin @fossasia/linux-devs

**Comment by abishekvashok** [comment]
Vote against it.

**Comment by rahulkumaran** [comment]
@meets2tarun @mariobehling @tabesin any updates on this?

**Comment by meets2tarun** [comment]
Go for it @rahulkumaran
